Source/URLRequestConvertible.swift

```diff
@@ -43,35 +43,108 @@ extension URLRequest: URLRequestConvertible {
 
 extension URLRequest {
     public init(path: String, baseURL: URL,
```

Comment:
It would be good to have the old version as well, and mark it as deprecated for an easy transition to the new API